src/commands/test.js

```diff
@@ -2,23 +2,38 @@ const karma = require("karma");
 const Server = karma.Server;
 const runner = karma.runner;
 const karmaConfig = require("../config/karma-config").default;
-
-const config = karmaConfig;
+const spawn = require("cross-spawn").spawn;
+const mocha = require("mocha");
+const path = require("path");
+const CWD = process.cwd();
 
 module.exports = function (options) {
+  // override to run tests in Node without Karma/Webpack
+  if (options.mochaOnly) {
+    const helperPath = path.join(__dirname, "..", "lib", "testHelperMocha.js");
+    const testPath = `${CWD}/test/**/*.js`;
```

Comment:
If we want to be consistent with our karma test helper, we'd use `*.test.js` as opposed to `*.js` at the end here: https://github.com/TrueCar/gluestick/blob/develop/src/lib/testHelper.js#L11